### ISSUE TITLE
Cannot read properties of undefined (reading 'Pe')

### ISSUE THREAD

**Issue by jamesozzie** [google/site-kit-wp#4235]
## Bug Description

There have [been reports](https://wordpress.org/plugins/health-check/) of the below dashboard error appearing for 2 users. 

```
Cannot read properties of undefined (reading 'Pe')

    in GoogleChart
    in ContextProvider
    in Chart
    in div
    in GoogleChart
    in div
    in Sparkline
    in div
    in div
    in DataBlock
    in div
    in DashboardImpressionsWidget
    in WhenSearchConsoleActive(DashboardImpressionsWidget)
    in div
    in div
    in Widget
    in WithWidgetSlug(Widget)
    in WidgetRenderer
    in div
    in Cell
    in WidgetCellWrapper
    in div
    in Row
    in div
    in Grid
    in div
    in Cell
    in div
    in Row
    in div
    in div
    in Grid
    in WidgetAreaRenderer
    in div
    in WidgetContextRenderer
    in DashboardApp
    in GoogleSitekitDashboard
    in RestoreSnapshots
    in ErrorHandler
    in Root
```

https://wordpress.org/support/topic/plugin-error-182/ | **Open** | Awaiting SH info
https://wordpress.org/support/topic/sitekit-error-please-fix-your-app/ | **Open** | Awaiting SH info

Troubleshooting and third party conflict checks ongoing. 

## Additional Context

 - PHP Version: 
 - Plugin Version [e.g. 22]

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by bethanylang** [comment]
Still awaiting additional insights and SH from users; will test when more info is available.

**Comment by bethanylang** [comment]
@felixarntz Reassigning to you for now and moving to L2.

**Comment by jamesozzie** [comment]
I've been able to recreate this today on one site. This particular site is on a hosting platform that has a very restrictive Firewall and from testing doesn't offer great performance. 

On occasions this site often times out when performing AMP validation and on occasions the Site Health status (below) displays that the site cannot perform loopback requests. 

![image](https://user-images.githubusercontent.com/41326532/140741125-bd02e562-4041-442d-a0cc-029a2a6fb64d.png)


<details>

<summary>
Console Errors
</summary>

```

JQMIGRATE: Migrate is installed, version 3.3.2
jsapi_compiled_default_module.js:73 Uncaught TypeError: b.createScript is not a function
    at gvjs_vf.ml (jsapi_compiled_default_module.js:73)
    at jsapi_compiled_default_module.js:73
jsapi_compiled_default_module.js:75 Uncaught TypeError: b.createScriptURL is not a function
    at gvjs_Ef (jsapi_compiled_default_module.js:75)
    at jsapi_compiled_ui_module.js:267
googlesitekit-vendor.09a03314e181c3acf88e.js:1 TypeError: Cannot read properties of undefined (reading 'Pe')
    at gvjs_Tn (jsapi_compiled_default_module.js:436)
    at gvjs_Q.gvjs_Sn (jsapi_compiled_default_module.js:435)
    at new gvjs_Q (jsapi_compiled_default_module.js:504)
    at GoogleChart.componentDidMount (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at ya (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at t.unstable_runWithPriority (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at Mr (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at Yj (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at Lj (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at googlesitekit-vendor.09a03314e181c3acf88e.js:1
Pi @ googlesitekit-vendor.09a03314e181c3acf88e.js:1
googlesitekit-module.3076f56f530551914e2a.js:21 Caught an error: TypeError: Cannot read properties of undefined (reading 'Pe')
    at gvjs_Tn (jsapi_compiled_default_module.js:436)
    at gvjs_Q.gvjs_Sn (jsapi_compiled_default_module.js:435)
    at new gvjs_Q (jsapi_compiled_default_module.js:504)
    at GoogleChart.componentDidMount (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at ya (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at t.unstable_runWithPriority (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at Mr (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at Yj (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at Lj (googlesitekit-vendor.09a03314e181c3acf88e.js:1)
    at googlesitekit-vendor.09a03314e181c3acf88e.js:1 Object
value @ googlesitekit-module.3076f56f530551914e2a.js:21
DevTools failed to load source map: Could not load content for https://beijing.ie/wp-content/plugins/pwa/wp-includes/js/workbox-v5.1.4/workbox-precaching.prod.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://beijing.ie/wp-content/plugins/pwa/wp-includes/js/workbox-v5.1.4/workbox-background-sync.prod.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://beijing.ie/wp-content/plugins/pwa/wp-includes/js/workbox-v5.1.4/workbox-routing.prod.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://beijing.ie/wp-content/plugins/pwa/wp-includes/js/workbox-v5.1.4/workbox-strategies.prod.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://beijing.ie/wp-content/plugins/pwa/wp-includes/js/workbox-v5.1.4/workbox-navigation-preload.prod.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://beijing.ie/wp-content/plugins/pwa/wp-includes/js/workbox-v5.1.4/workbox-core.prod.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```

</details>



<details>

<summary>
Site Health info
</summary>

 
```

### wp-core ###

version: 5.7.3
site_language: en_US
user_language: en_US
permalink: /%postname%/
https_status: true
user_registration: 0
default_comment_status: closed
multisite: false
user_count: 2
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /home/ccreygrp/public_html/beijing.ie
wordpress_size: 44.93 MB (47116652 bytes)
uploads_path: /home/ccreygrp/public_html/beijing.ie/wp-content/uploads
uploads_size: 6.06 MB (6357718 bytes)
themes_path: /home/ccreygrp/public_html/beijing.ie/wp-content/themes
themes_size: 23.29 MB (24421609 bytes)
plugins_path: /home/ccreygrp/public_html/beijing.ie/wp-content/plugins
plugins_size: 82.41 MB (86410859 bytes)
database_size: 19.35 MB (20288160 bytes)
total_size: 176.04 MB (184594998 bytes)

### wp-active-theme ###

name: Twenty Twenty-One (twentytwentyone)
version: 1.4
author: the WordPress team
author_website: https://wordpress.org/
parent_theme: none
theme_features: core-block-patterns, automatic-feed-links, title-tag, post-formats, post-thumbnails, menus, html5, custom-logo, customize-selective-refresh-widgets, wp-block-styles, align-wide, editor-styles, editor-style, editor-font-sizes, custom-background, editor-color-palette, editor-gradient-presets, responsive-embeds, custom-line-height, experimental-link-color, custom-spacing, custom-units, amp, widgets
theme_path: /home/ccreygrp/public_html/beijing.ie/wp-content/themes/twentytwentyone

### wp-themes-inactive (6) ###

AMPFace: version: 1.6.0, author: James Ozzie Osborne
Astra: version: 3.2.0, author: Brainstorm Force (latest version: 3.7.5)
Breek Child: author: EstudioPatagon, version: (undefined)
Breek: version: 3.6.6, author: EstudioPatagon
ExS: version: 1.7.1, author: the ExS team (latest version: 1.8.0)
Twenty Twenty: version: 1.7, author: the WordPress team (latest version: 1.8)

### wp-mu-plugins (1) ###

Health Check Troubleshooting Mode: author: (undefined), version: 1.7.2

### wp-plugins-active (8) ###

Advanced Custom Fields PRO: version: 5.10.2, author: Delicious Brains
AMP: version: 2.2.0-alpha-20210924T010757Z-73cfc46, author: AMP Project Contributors
Breek Theme Functions: version: 3.6.6, author: Estudio Patagon
Contact Form 7: version: 5.4.2, author: Takayuki Miyoshi (latest version: 5.5.2)
Floating Button Lite: version: 5.1, author: Wow-Company
Health Check & Troubleshooting: version: 1.4.5, author: The WordPress.org community
RegistrationMagic: version: 5.0.1.4, author: Registrationmagic (latest version: 5.0.1.6)
Site Kit by Google: version: 1.42.0, author: Google (latest version: 1.44.0)

### wp-plugins-inactive (5) ###

Ad Inserter: version: 2.7.4, author: Igor Funa (latest version: 2.7.5)
Akismet Anti-Spam: version: 4.1.10, author: Automattic (latest version: 4.2.1)
Duplicator: version: 1.4.2, author: Snap Creek (latest version: 1.4.3)
Jetpack: version: 9.9, author: Automattic (latest version: 10.3)
Web Stories: version: 1.8.1, author: Google (latest version: 1.14.0)

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Not available
imagemagick_version: Not available
gd_version: bundled (2.1.0 compatible)
ghostscript_version: unknown

### wp-server ###

server_architecture: Linux 3.10.0-962.3.2.lve1.5.52.el7.x86_64 x86_64
httpd_software: Apache
php_version: 7.3.32 64bit
php_sapi: fpm-fcgi
max_input_variables: 1000
time_limit: 90
memory_limit: 256M
max_input_time: 90
upload_max_size: 1024M
php_post_max_size: 1024M
curl_version: 7.79.1 OpenSSL/1.1.1l
suhosin: false
imagick_availability: false
server-headers: 
	server: nginx
	date: Mon, 08 Nov 2021 12:22:09 GMT
	content-type: text/html; charset=UTF-8
	vary: Accept-Encoding
	pragma: no-cache
	expires: Wed, 11 Jan 1984 05:00:00 GMT
	cache-control: no-cache, must-revalidate, max-age=0
	link: <https://beijing.ie/wp-json/>; rel="https://api.w.org/"
	strict-transport-security: max-age=31536000; includeSubDomains; preload
	x-frame-options: SAMEORIGIN
	x-content-type-options: Array
	x-xss-protection: Array
	x-nginx-upstream-cache-status: BYPASS
	x-server-powered-by: Engintron
	content-encoding: gzip
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 10.3.31-MariaDB-cll-lve
client_version: mysqlnd 5.0.12-dev - 20150407 - $Id: 7cc7cc96e675f6d72e5cf0f267f48e167c2abb23 $

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /home/ccreygrp/public_html/beijing.ie/wp-content
WP_PLUGIN_DIR: /home/ccreygrp/public_html/beijing.ie/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable
mu-plugins: writable

### google-site-kit ###

version: 1.42.0
php_version: 7.3.32
wp_version: 5.7.3
reference_url: https://beijing.ie
amp_mode: secondary
site_status: connected-site
user_status: authenticated
verification_status: verified-non-site-kit
connected_user_count: 1
active_modules: site-verification, search-console, adsense, analytics, analytics-4, tagmanager
required_scopes: 
	openid: ✅
	https://www.googleapis.com/auth/userinfo.profile: ✅
	https://www.googleapis.com/auth/userinfo.email: ✅
	https://www.googleapis.com/auth/siteverification: ✅
	https://www.googleapis.com/auth/webmasters: ✅
	https://www.googleapis.com/auth/adsense.readonly: ✅
	https://www.googleapis.com/auth/analytics.readonly: ✅
	https://www.googleapis.com/auth/tagmanager.readonly: ✅
capabilities: 
	googlesitekit_authenticate: ✅
	googlesitekit_setup: ✅
	googlesitekit_view_posts_insights: ✅
	googlesitekit_view_dashboard: ✅
	googlesitekit_view_module_details: ✅
	googlesitekit_manage_options: ✅
enabled_features: 
	ga4setup: ✅
	ideaHubModule: ⭕
	serviceSetupV2: ⭕
	swgModule: ⭕
	userInput: ⭕
	unifiedDashboard: ⭕
search_console_property: https://beijing.ie/
adsense_account_id: pub-346•••••••••••••
adsense_client_id: ca-pub-346•••••••••••••
adsense_account_status: approved
adsense_use_snippet: yes
adsense_web_stories_adunit_id: none
analytics_account_id: 1773•••••
analytics_property_id: UA-1773•••••••
analytics_profile_id: 2283•••••
analytics_use_snippet: yes
analytics_4_property_id: 2765637••
analytics_4_web_data_stream_id: 2665••••••
analytics_4_measurement_id: G-1W••••••••
analytics_4_use_snippet: yes
tagmanager_account_id: none
tagmanager_container_id: none
tagmanager_amp_container_id: none
tagmanager_use_snippet: yes

### amp_wp ###

amp_slug_query_var: amp
amp_slug_defined_late: false
amp_mode_enabled: transitional
amp_reader_theme: legacy
amp_templates_enabled: post, page, attachment, is_singular, is_home, is_archive, is_author, is_date, is_search, is_404, is_category, is_tag
amp_serve_all_templates: true
amp_css_transient_caching_disabled: false
amp_css_transient_caching_threshold: 5000 transients per day
amp_css_transient_caching_sampling_range: 14 days
amp_css_transient_caching_transient_count: 127
amp_css_transient_caching_time_series: 
	20211026: 151
	20211027: 160
	20211028: 160
	20211029: 147
	20211030: 147
	20211031: 147
	20211101: 141
	20211102: 137
	20211103: 145
	20211104: 154
	20211105: 159
	20211106: 160
	20211107: 128
	20211108: 127
amp_libxml_version: 2.9.7
```

</details>

**Comment by aaemnnosttv** [comment]
@jamesozzie @bethanylang I've confirmed the source of the conflict here to be from the [RegistrationMagic – Custom Registration Forms, User Registration and User Login Plugin](https://wordpress.org/plugins/custom-registration-form-builder-with-submission-manager/) plugin which is enqueuing the Google charts loader and the latest version of the charts library on **all admin screens**.

![image](https://user-images.githubusercontent.com/1621608/140923151-5ccbab78-12a9-49e3-8986-6657b042097a.png)

![image](https://user-images.githubusercontent.com/1621608/140924130-e1c4431b-8602-464a-9d43-1e237311830b.png)

This conflicts with the specific version of the charts library that we load on the page (`49`) 
![image](https://user-images.githubusercontent.com/1621608/140923334-b418b6db-748f-428d-b94d-96a6a743d14f.png)

Disabling the RegistrationMagic plugin fixes the problem. Unfortunately it looks like they'll need to release an update to fix this.

**Comment by jamesozzie** [comment]
@aaemnnosttv Thanks for checking this. I'll open a support topic with the plugin developer and check with any impacted users.

**Comment by bethanylang** [comment]
@jamesozzie Any update here? Thanks!

**Comment by jamesozzie** [comment]
@bethanylang I'm awaiting an update from the Registration Magic team. I've requested an update now. 

I also noticed one of our users opened a support topic [on their support forums](https://wordpress.org/support/topic/google-site-kit-cannot-read-properties-of-undefined-reading-pe/).

**Comment by jamesozzie** [comment]
The Registration Magic plugin was updated however there is no fix for this just yet. I've requested a status update from my open support ticket on their website.

**Comment by jamesozzie** [comment]
The RegistrationMagic team have confirmed they'll a fix for this in place in an upcoming release. As there's no action required on the Site Kit site I'm closing this issue while following up with impacted users.

**Comment by jamesozzie** [comment]
Just noting here that the errors that the specific error that is output may vary, with the below appearing:
- `undefined is not an object (evaluating 'a.ft.Pe')` ([related support topic](https://wordpress.org/support/topic/site-kit-encountered-an-error-58/))
- A cross-origin error was thrown. React doesn't have access to the actual error object in development. See https://fb.me/react-crossorigin-error for more information. (This occurred on a support test site)

![image](https://user-images.githubusercontent.com/41326532/147576561-8e83f64a-7052-43c8-b65a-44b2dead786e.png)

![image](https://user-images.githubusercontent.com/41326532/147576654-287e66ca-fd0e-4409-93ed-ca3379d2575b.png)


<details><summary>
Console Errors
</summary>

```
JQMIGRATE: Migrate is installed with logging active, version 3.3.2
jquery-migrate.js?ver=3.3.2:100 JQMIGRATE: jQuery.expr[':'] is deprecated; use jQuery.expr.pseudos
migrateWarn @ jquery-migrate.js?ver=3.3.2:100
jquery-migrate.js?ver=3.3.2:102 console.trace
migrateWarn @ jquery-migrate.js?ver=3.3.2:102
jquery-migrate.js?ver=3.3.2:100 JQMIGRATE: jQuery.isArray is deprecated; use Array.isArray
migrateWarn @ jquery-migrate.js?ver=3.3.2:100
jquery-migrate.js?ver=3.3.2:102 console.trace
migrateWarn @ jquery-migrate.js?ver=3.3.2:102
jquery-migrate.js?ver=3.3.2:100 JQMIGRATE: jQuery.isFunction() is deprecated
migrateWarn @ jquery-migrate.js?ver=3.3.2:100
jquery-migrate.js?ver=3.3.2:102 console.trace
migrateWarn @ jquery-migrate.js?ver=3.3.2:102
react-dom.js?ver=16.13.1:25129 Download the React DevTools for a better development experience: https://fb.me/react-devtools
react-dom.development.js?61bb:27773 Download the React DevTools for a better development experience: https://fb.me/react-devtools
react-dom.development.js?1925:24994 Download the React DevTools for a better development experience: https://fb.me/react-devtools
jquery-migrate.js?ver=3.3.2:100 JQMIGRATE: jQuery.fn.mouseenter() event shorthand is deprecated
migrateWarn @ jquery-migrate.js?ver=3.3.2:100
jquery-migrate.js?ver=3.3.2:102 console.trace
migrateWarn @ jquery-migrate.js?ver=3.3.2:102
jquery-migrate.js?ver=3.3.2:100 JQMIGRATE: jQuery.fn.mouseleave() event shorthand is deprecated
migrateWarn @ jquery-migrate.js?ver=3.3.2:100
jquery-migrate.js?ver=3.3.2:102 console.trace
migrateWarn @ jquery-migrate.js?ver=3.3.2:102
jquery-migrate.js?ver=3.3.2:100 JQMIGRATE: jQuery.fn.click() event shorthand is deprecated
migrateWarn @ jquery-migrate.js?ver=3.3.2:100
jquery-migrate.js?ver=3.3.2:102 console.trace
migrateWarn @ jquery-migrate.js?ver=3.3.2:102
jquery-migrate.js?ver=3.3.2:100 JQMIGRATE: jQuery.fn.submit() event shorthand is deprecated
migrateWarn @ jquery-migrate.js?ver=3.3.2:100
jquery-migrate.js?ver=3.3.2:102 console.trace
migrateWarn @ jquery-migrate.js?ver=3.3.2:102
jquery-migrate.js?ver=3.3.2:100 JQMIGRATE: jQuery.fn.change() event shorthand is deprecated
migrateWarn @ jquery-migrate.js?ver=3.3.2:100
jquery-migrate.js?ver=3.3.2:102 console.trace
migrateWarn @ jquery-migrate.js?ver=3.3.2:102
jsapi_compiled_default_module.js:73 Uncaught TypeError: b.createScript is not a function
    at gvjs_vf.ml (www.gstatic.com/charts/49/js/jsapi_compiled_default_module.js:73)
    at www.gstatic.com/charts/49/js/jsapi_compiled_default_module.js:73
jsapi_compiled_default_module.js:75 Uncaught TypeError: b.createScriptURL is not a function
    at gvjs_Ef (www.gstatic.com/charts/49/js/jsapi_compiled_default_module.js:75)
    at www.gstatic.com/charts/49/js/jsapi_compiled_ui_module.js:267
jsapi_compiled_default_module.js:436 Uncaught TypeError: Cannot read properties of undefined (reading 'Pe')
    at gvjs_Tn (www.gstatic.com/charts/49/js/jsapi_compiled_default_module.js:436)
    at gvjs_Q.gvjs_Sn (www.gstatic.com/charts/49/js/jsapi_compiled_default_module.js:435)
    at new gvjs_Q (www.gstatic.com/charts/51/js/jsapi_compiled_default_module.js:504)
    at GoogleChart.componentDidMount (webpack-internal:///./node_modules/react-google-charts/dist/index.esm.js:632)
    at commitLifeCycles (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:22101)
    at commitLayoutEffects (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:25344)
    at HTMLUnknownElement.callCallback (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:336)
    at Object.invokeGuardedCallbackDev (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:385)
    at invokeGuardedCallback (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:440)
    at commitRootImpl (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:25082)
react-dom.development.js?61bb:21843 The above error occurred in the <GoogleChart> component:
    in GoogleChart (created by Chart)
    in ContextProvider (created by Chart)
    in Chart (created by GoogleChart)
    in div (created by GoogleChart)
    in GoogleChart (created by Sparkline)
    in div (created by Sparkline)
    in Sparkline (created by DashboardSearchVisitorsWidget)
    in div (created by DataBlock)
    in div (created by DataBlock)
    in DataBlock (created by DashboardSearchVisitorsWidget)
    in DashboardSearchVisitorsWidget (created by WhenAnalyticsActive(DashboardSearchVisitorsWidget))
    in WhenAnalyticsActive(DashboardSearchVisitorsWidget) (created by WidgetRenderer)
    in div (created by Widget)
    in div (created by Widget)
    in Widget (created by WithWidgetSlug(Widget))
    in WithWidgetSlug(Widget) (created by WidgetRenderer)
    in WidgetRenderer (created by WidgetAreaRenderer)
    in div (created by Cell)
    in Cell (created by WidgetCellWrapper)
    in WidgetCellWrapper (created by WidgetAreaRenderer)
    in div (created by Row)
    in Row (created by WidgetAreaRenderer)
    in div (created by ForwardRef)
    in ForwardRef (created by WidgetAreaRenderer)
    in div (created by Cell)
    in Cell (created by WidgetAreaRenderer)
    in div (created by Row)
    in Row (created by WidgetAreaRenderer)
    in div (created by WidgetAreaRenderer)
    in div (created by ForwardRef)
    in ForwardRef (created by WidgetAreaRenderer)
    in WidgetAreaRenderer (created by WidgetContextRenderer)
    in div (created by WidgetContextRenderer)
    in WidgetContextRenderer (created by DashboardApp)
    in DashboardApp (created by DashboardEntryPoint)
    in DashboardEntryPoint
    in RestoreSnapshots (created by Root)
    in ErrorHandler (created by Root)
    in Root

React will try to recreate this component tree from scratch using the error boundary you provided, ErrorHandler.
logCapturedError @ react-dom.development.js?61bb:21843
index.js?68d0:55 Caught an error: Error: A cross-origin error was thrown. React doesn't have access to the actual error object in development. See https://fb.me/react-crossorigin-error for more information.
    at Object.invokeGuardedCallbackDev (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:396)
    at invokeGuardedCallback (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:440)
    at commitRootImpl (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:25082)
    at unstable_runWithPriority (webpack-internal:///./node_modules/scheduler/cjs/scheduler.development.js:697)
    at runWithPriority$2 (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:12149)
    at commitRoot (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:24922)
    at finishSyncRender (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:24329)
    at performSyncWorkOnRoot (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:24307)
    at eval (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:12199)
    at unstable_runWithPriority (webpack-internal:///./node_modules/scheduler/cjs/scheduler.development.js:697) ObjectcomponentStack: "\n    in GoogleChart (created by Chart)\n    in ContextProvider (created by Chart)\n    in Chart (created by GoogleChart)\n    in div (created by GoogleChart)\n    in GoogleChart (created by Sparkline)\n    in div (created by Sparkline)\n    in Sparkline (created by DashboardSearchVisitorsWidget)\n    in div (created by DataBlock)\n    in div (created by DataBlock)\n    in DataBlock (created by DashboardSearchVisitorsWidget)\n    in DashboardSearchVisitorsWidget (created by WhenAnalyticsActive(DashboardSearchVisitorsWidget))\n    in WhenAnalyticsActive(DashboardSearchVisitorsWidget) (created by WidgetRenderer)\n    in div (created by Widget)\n    in div (created by Widget)\n    in Widget (created by WithWidgetSlug(Widget))\n    in WithWidgetSlug(Widget) (created by WidgetRenderer)\n    in WidgetRenderer (created by WidgetAreaRenderer)\n    in div (created by Cell)\n    in Cell (created by WidgetCellWrapper)\n    in WidgetCellWrapper (created by WidgetAreaRenderer)\n    in div (created by Row)\n    in Row (created by WidgetAreaRenderer)\n    in div (created by ForwardRef)\n    in ForwardRef (created by WidgetAreaRenderer)\n    in div (created by Cell)\n    in Cell (created by WidgetAreaRenderer)\n    in div (created by Row)\n    in Row (created by WidgetAreaRenderer)\n    in div (created by WidgetAreaRenderer)\n    in div (created by ForwardRef)\n    in ForwardRef (created by WidgetAreaRenderer)\n    in WidgetAreaRenderer (created by WidgetContextRenderer)\n    in div (created by WidgetContextRenderer)\n    in WidgetContextRenderer (created by DashboardApp)\n    in DashboardApp (created by DashboardEntryPoint)\n    in DashboardEntryPoint\n    in RestoreSnapshots (created by Root)\n    in ErrorHandler (created by Root)\n    in Root"[[Prototype]]: Object
componentDidCatch @ index.js?68d0:55
/wp-json/google-site-kit/v1/modules/idea-hub/data/saved-ideas?timestamp=false&_locale=user:1 Failed to load resource: the server responded with a status of 403 ()
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:saved-ideas type:modules identifier:idea-hub error:"Permission denied"
_callee$ @ index.js?2c09:187
/wp-json/google-site-kit/v1/modules/search-console/data/searchanalytics?startDate=2021-11-30&endDate=2021-12-27&dimensions=query&limit=10&_locale=user:1 Failed to load resource: the server responded with a status of 403 ()
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:searchanalytics type:modules identifier:search-console error:"User does not have sufficient permission for site 'https://plastiskip.com/'. See also: https://support.google.com/webmasters/answer/2451999."
_callee$ @ index.js?2c09:187
/wp-json/google-site-kit/v1/modules/analytics/data/report?metrics%5B0%5D%5Bexpression%5D=ga%3Ausers&dimensions%5B0%5D%5Bname%5D=ga%3AchannelGrouping&startDate=2021-11-30&endDate=2021-12-27&compareStartDate=2021-11-02&compareEndDate=2021-11-29&orderby%5BfieldName%5D=ga%3Ausers&orderby%5BsortOrder%5D=DESCENDING&limit=6&_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:report type:modules identifier:analytics error:"The response is not a valid JSON response."
_callee$ @ index.js?2c09:187
/wp-json/google-site-kit/v1/modules/search-console/data/searchanalytics?dimensions=date&startDate=2021-11-02&endDate=2021-12-27&_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
/wp-json/google-site-kit/v1/modules/analytics/data/report?metrics%5B0%5D%5Bexpression%5D=ga%3AgoalCompletionsAll&metrics%5B0%5D%5Balias%5D=Goal%20Completions&dimensions%5B0%5D%5Bname%5D=ga%3Adate&compareStartDate=2021-11-02&compareEndDate=2021-11-29&startDate=2021-11-30&endDate=2021-12-27&_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
/wp-json/google-site-kit/v1/modules/analytics/data/report?metrics%5B0%5D%5Bexpression%5D=ga%3Ausers&metrics%5B0%5D%5Balias%5D=Total%20Users&startDate=2021-11-30&endDate=2021-12-27&url=&compareStartDate=2021-11-02&compareEndDate=2021-11-29&_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:searchanalytics type:modules identifier:search-console error:"The response is not a valid JSON response."
_callee$ @ index.js?2c09:187
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:report type:modules identifier:analytics error:"The response is not a valid JSON response."
_callee$ @ index.js?2c09:187
/wp-json/google-site-kit/v1/modules/analytics/data/goals?_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:report type:modules identifier:analytics error:"The response is not a valid JSON response."
_callee$ @ index.js?2c09:187
/wp-json/google-site-kit/v1/modules/idea-hub/data/new-ideas?timestamp=false&_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
/wp-json/google-site-kit/v1/modules/analytics/data/report?metrics%5B0%5D%5Bexpression%5D=ga%3Apageviews&metrics%5B0%5D%5Balias%5D=Pageviews&dimensions%5B0%5D%5Bname%5D=ga%3ApagePath&startDate=2021-11-30&endDate=2021-12-27&orderby%5B0%5D%5BfieldName%5D=ga%3Apageviews&orderby%5B0%5D%5BsortOrder%5D=DESCENDING&limit=10&_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
/wp-json/google-site-kit/v1/modules/pagespeed-insights/data/pagespeed?strategy=mobile&url=https%3A%2F%2Fplastiskip.com%2F&_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:goals type:modules identifier:analytics error:"The response is not a valid JSON response."
_callee$ @ index.js?2c09:187
/wp-json/google-site-kit/v1/modules/pagespeed-insights/data/pagespeed?strategy=desktop&url=https%3A%2F%2Fplastiskip.com%2F&_locale=user:1 Failed to load resource: the server responded with a status of 504 ()
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:new-ideas type:modules identifier:idea-hub error:"The response is not a valid JSON response."
_callee$ @ index.js?2c09:187
index.js?2c09:187 Google Site Kit API Error method:GET datapoint:report type:modules identifier:analytics error:"The response is not a valid JSON response."
_callee$ @ index.js?2c09:187
2index.js?2c09:187 Google Site Kit API Error method:GET datapoint:pagespeed type:modules identifier:pagespeed-insights error:"The response is not a valid JSON response."
_callee$ @ index.js?2c09:187
/wp-admin/admin-ajax.php:1 Failed to load resource: the server responded with a status of 504 ()
```

</details>

**Comment by jamesozzie** [comment]
Just a quick note here. Registration Magic have released an update to their plugin that includes a fix for this conflict, version 5.0.2.2. 

For any users who encountered this error please update to this latest version of Registration Magic. 

If you previously used the mini plugin proposed above as a temporary solution, you can now deactivate and uninstall this as it's no longer required.
